src/lib/setup/refresh-xkcd.ts

```diff
@@ -0,0 +1,11 @@
+import { refreshComicsFromRemote } from '#lib/utilities/xkcd';
+import { job } from 'cron';
+
+export const xkcdRefresh = job({
+	// cronTime: '0 0 12,13 * * 1-5',
+	cronTime: '0 35 14 * * 1-5',
+	onTick: refreshComicsFromRemote,
+	start: true,
+	unrefTimeout: true,
+	timeZone: 'Etc/UTC'
```

Comment:
ArchId server has UTC+2 (CEST) (`Europe/Berlin`)

Reply:
The GH action runs on UTC.